src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -1074,6 +1075,31 @@ public async Task CanPatchMoreThan10AttributesInAnItemWithVariables()
             Assert.AreEqual(patchResponse.GetProperty("moons").ToString(), JsonSerializer.Serialize(update.moons));
         }
 
+        [TestMethod]
+        [DataRow("en-US")]
+        [DataRow("en-DE")]
+        public async Task CanCreateItemWithCultureInvariant(string cultureInfo)
+        {
+            CultureInfo ci = new(cultureInfo);
+            CultureInfo.DefaultThreadCurrentCulture = ci;
```

Comment:
Comment would help, this defines the culture of the DAB "host" machine or "thread." 

So in this test a double value `10.15` (en-US culture) would be persisted to database as "cultureInfo.InvariantCulture" which wouldn't change the value -> stored as `10.15`. but then when response is returned, because the DefaultThreadCulture is `en-DE`, the value is converted in the response to match the host or client's culture?

Reply:
No, In response, value won't be converted back to host/client culture, thats what I am asserting here by putting "expected value" as cultural invariant.